src/components/Description/index.jsx

```diff
@@ -1,11 +1,11 @@
 
 export default function Description(){
-    return (<div className={"container m-auto px-[5rem]"}>
-                    <div className={"mx-[5rem] min-w-full"}>
+    return (<div className={"container text-center sm:text-left sm:m-auto px-[5rem]"}>
+                    <div className={"sm:mx-[5rem] min-w-full"}>
                         <h1>Description</h1>
                     </div>
-                    <div className={"ml-[10rem] min-w-fit"}>
-                        <p>Flourish! is an annually held conference intendedc to introduce both CS and non-CS, technical and non-technical majors to the world of open source through speakers, demos, workshops, sponsors, job opportunities, and other events! We show off hardware, software, business, entrepreneurship, finance, and more aspects of open source. We want to make this project bigger and bigger year over year and bring in new students. Our previous conferences focused on a small number of students in a technical software direction, but we are looking to change that!</p>
+                    <div className={"sm:ml-[10rem] min-w-fit"}>
+                        <body><i>Flourish!</i> is an annually held conference intended to introduce both CS and non-CS, technical and non-technical majors to the world of open source through speakers, demos, workshops, sponsors, job opportunities, and other events! We show off hardware, software, business, entrepreneurship, finance, and more aspects of open source. We want to make this project bigger and bigger year over year and bring in new students. Our previous conferences focused on a small number of students in a technical software direction, but we are looking to change that!</body>
```

Comment:
This should be p instead of body, also I don't believe Flourish! should be italicized.